### ISSUE TITLE
Tab navigation skips tutorial nudge button

### ISSUE THREAD

**Issue by lenacohen** [EFForg/privacybadger#3139]
The "Learn how Privacy Badger protects your privacy" button is not accessible via tab navigation (without a screenreader). It is accessible via tab navigation when using a screenreader (VoiceOver on Mac), but not if you're doing keyboard navigation without a screenreader.
<img width="558" height="409" alt="Image" src="https://github.com/user-attachments/assets/638314e5-7759-47c9-9272-1b20873d7a31" />

To see this button:
- Install Privacy Badger
- Close the welcome page tab without scrolling down or clicking the "Take the Tour" button
- Open Privacy Badger's popup on another page
